packages/document-schema/model-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@palantir/pack.document-schema.model-types",
-  "version": "0.0.1",
+  "version": "0.0.1-beta.1",
```

Comment:
from osdk-ts, it seems that packages are held at `-beta.x` versions "at rest", and only get proper version on the publish commits, so setting this to match that.